subsys/bluetooth/host/adv.c

```diff
@@ -430,6 +430,20 @@ static inline bool ad_has_name(const struct bt_data *ad, size_t ad_len)
 	return false;
 }
 
+static inline bool ad_is_limited(const struct bt_data *ad, size_t ad_len)
```

Comment:
```suggestion
static bool ad_is_limited(const struct bt_data *ad, size_t ad_len)
```
https://www.kernel.org/doc/html/v4.10/process/coding-style.html#the-inline-disease